### DISCUSSION TITLE
Vim Colorschemes don't match?

### DISCUSSION THREAD

**Discussion by hsover** [kovidgoyal/kitty#5822] [Q&A]
Hi, I've installed so many themes and I've been trying to work out why none of them look great until I realized its because the colors don't match.

Expected: https://user-images.githubusercontent.com/42694704/196934147-0704f7ec-456f-4a07-b44f-ac99896c7b22.png
Actual: https://i.imgur.com/1rVN5YO.png

I'm using KItty on Arch Linux. As you can see, the colors seem very washed out compared to the showcased picture.

I disabled all of my kitty settings to see if the result is the same and it is. Can somebody tell me what's wrong?

**Comment by kovidgoyal** [comment]
Dont match with what? And when asking questions provide relevant information, kitty version, vim version, the color scheme you are trying. You need to make sure your color scheme is using truecolor on the terminal.

**Comment by AprilArcus** [comment]
The colors in the "expected" screenshot appear excessively vivid because they are being rendered into the display's device-native colorspace, not a calibrated colorspace like sRGB or Display P3. Another way of saying this is that the sRGB red is (255,0,0), but modern displays are capable of making even redder reds than that, and can render colors outside the sRGB gamut which can't be represented as hexadecimal triples. The reddest red in P3 corresponds to an sRGB value of (279, -58, -38).

<img width="680" alt="image" src="https://user-images.githubusercontent.com/2045543/217783928-504a83a2-60af-4e0e-bd47-57c2f4dfe5c3.png">

Without color calibration, the sRGB values of your color theme will be "stretched" to your device's gamut. If the color theme author asks for the reddest red on his display, you'll receive the reddest red on *your* display, which might be much more saturated than the developer intended. This is why by default, Kitty and most well-behaved terminal emulators and text editors render colors into the sRGB colorspace for consistency across devices. 

However, some people prefer the excessively vivid colors created by stretching sRGB colors into P3 or device-native space. You can achieve this in Kitty by setting the `macos_colorspace default` config, which disables color calibration in Kitty. Do note that this will also impact the rendering of PNG images, which will then differ from the calibrated presentation inside e.g. a web browser.
